wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsDFUClient.java

```diff
@@ -97,6 +98,15 @@ public static HPCCWsDFUClient get(Connection connection)
     private boolean               verbose               = false;
     private WsDFUClientSoapProxyWrapper soapproxywrapper = null;
 
+    public boolean hasInitError()
+    {
+        return !initErrMessage.isEmpty();
+    }
+
+    public String getInitError()
+    {
+        return initErrMessage;
```

Comment:
Should the logic check if initErrMessage is null and returning an empty string be moved here? Or would it be better to make initErrMessage default to an empty String to avoid null check all together?

Reply:
setting to empty string